### PR TITLE
Suggestion for allowing also XPointer Framework Fragment IDs

### PR DESCRIPTION
**Pull request by akeranen**

### CODE REVIEW

File: draft-ietf-core-senml.md [hunk 1]
```diff
@@ -983,6 +993,12 @@ from 19th to the last:
 
   coap://example.com/temp#rec=3-5,10,19-*
 
+## Fragment Identification for the XML Format
+
+In addition to the SenML Fragment Identifiers described above, with 
+the XML SenML formats also the syntax defined in {{XPointerFramework}}
+can be used, as recommended by {{!RFC5703}}.
```
Comment (cabo) [inline draft-ietf-core-senml.md]: 7303, I presume?

File: draft-ietf-core-senml.md [hunk 2]
```diff
@@ -983,6 +993,12 @@ from 19th to the last:
 
   coap://example.com/temp#rec=3-5,10,19-*
 
+## Fragment Identification for the XML Format
+
+In addition to the SenML Fragment Identifiers described above, with 
+the XML SenML formats also the syntax defined in {{XPointerFramework}}
```
Comment (cabo) [inline draft-ietf-core-senml.md]: Does "XML SenML formats" include EXI?

File: draft-ietf-core-senml.md (hunk 1 above)
Comment (akeranen) [reply draft-ietf-core-senml.md]: Yes, fixed.

File: draft-ietf-core-senml.md (hunk 2 above)
Comment (akeranen) [reply draft-ietf-core-senml.md]: Good question. I suppose same fragment IDs work for EXI? If yes, could make it explicit here.

Comment (cabo) [reply draft-ietf-core-senml.md]: So see my PR to this PR...

Comment (cabo) [reply draft-ietf-core-senml.md]: #117, that is.

**Review by fluffy — APPROVED** [review]
I'm fine with this.

**Pull request merged**